src/ServiceControl.AcceptanceTests/MessageFailures/When_a_retry_for_a_failed_message_is_successful.cs

```diff
@@ -48,10 +48,9 @@ public void Should_show_up_as_resolved_when_doing_a_single_retry()
                 .Run(TimeSpan.FromMinutes(2));
 
             Assert.AreEqual(FailedMessageStatus.Resolved, failure.Status);
-            Assert.AreEqual(failure.UniqueMessageId, message.Id);
+            //Assert.AreEqual(failure.UniqueMessageId, message.Id);
```

Comment:
@johnsimons actually, it did work because the results go through [this transformer](https://github.com/Particular/ServiceControl/blob/1ea1d2d566cd32463a3941a2c92ac36f781f72cd/src/ServiceControl/CompositeViews/Messages/MessagesViewTransformer.cs#L46) which sets the `message.Id` to the `FailedMessage.UniqueMessageId`. 

It no longer works because there is a discrepancy between the `UniqueMessageId` generated for the failing message versus the successful retry. The failing messages uses the `MessageId` and the `FailedQ` header to generate it's `UniqueMessageId`. The successful retry does not have a `FailedQ` header any more so it uses `MessageId` and `ReplyToAddress`.

In version 1.26 of ServiceControl, the failed message would have been using the `ReplyToAddress` as well so these would have matched. In a broker based transport where the `ReplyToAddress` and the `FailedQ` are likely to match, this is going to be less of an issue.